frontend/src/features/update-workflow/ui/nodes/editableNodes.module.css

```
@@ -0,0 +1,217 @@
+.base {
+  font-family: figmaSans, "figmaSans Fallback", SF Pro Display, system-ui, helvetica;
```

Stylelint (17.9.0)

[error] Expected quotes around "SF Pro Display" (font-family-name-quotes)

(font-family-name-quotes)